spring-integration-core/src/test/java/org/springframework/integration/dsl/reactivestreams/ReactiveStreamsTests.java

```diff
@@ -124,7 +122,7 @@ public void testPollableReactiveFlow() throws Exception {
 								.map(v -> v.split(","))
 								.flatMapIterable(Arrays::asList)
 								.map(Integer::parseInt)
-								.<Message<Integer>>map(GenericMessage<Integer>::new)
+								.<Message<Integer>>map(GenericMessage::new)
 								.concatWith(this.pollablePublisher)
```

Comment:
Why remove the generic type here (and below). Works for me with it restored (and removes the warning from my IDE).

Reply:
OK. Will restore.
My picture in the IDEA looks like:
![image](https://user-images.githubusercontent.com/1071590/30122589-f6dcb1e8-92fc-11e7-98e3-fa83401dbd66.png)

So, that made me to go ahead and remove that gray code 😄

Reply:
I guess eclipse is not so smart 👎 😄